src/com/epam/Member.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.training;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+public class Member {
+    public Scanner input = new Scanner(System.in);
+    public Scanner input1 = new Scanner(System.in);
```

Comment:
don't see the reason why you have used 2 Scanner ? In my understanding one is enough, isn't it?